apollo-router/src/plugins/telemetry/config_new/conditions.rs

```diff
@@ -267,6 +268,44 @@ where
             Condition::False => false,
         }
     }
+
+    pub(crate) fn evaluate_response_field(&self, typed_value: &TypedValue, ctx: &Context) -> bool {
+        match self {
+            Condition::Eq(eq) => {
```

Comment:
Test needed for this fn

Reply:
[f631886](https://github.com/apollographql/router/pull/5215/commits/f63188657a458b741455b00adf4d787bf399ac18)